c2corg_ui/static/js/documentediting.js

```diff
@@ -536,31 +577,52 @@ app.DocumentEditingController.prototype.updateMaxSteps = function(waypointType)
  * @param {Object} outing
  * @private
  */
-
 app.DocumentEditingController.prototype.formatOuting_ = function(outing) {
-  // creating a new outing -> init locales
+  if (this.submit_) {
+    if (outing['locales'][0]['conditions_levels'][0]['level_place'].length > 0) {
```

Comment:
I think the formatOuting method is quite confusing.
As discussed, I would better see these low level conversions in the apiservice.
It may be too much for this already big PR, so feel free to try it later.